Assignment_10/SList.java

```diff
@@ -0,0 +1,40 @@
+package com.assignment.list;
+
+/**
+ * This is the main list class from which we can access list through list iterator.
+ * @param <T>
+ */
+public class SList<T>  {
+
+     Node<T> head=new Node<>(null);
+     SListIterator<T> iterator;
+
+    SList(){
+        iterator = new SListIterator<T>(head);
+    }
+    
+    public SListIterator<T> iterator()
+    {
+        return iterator;
+    }
+    
+    public String toString() {
+
+
+        SListIterator<T> it = this.iterator();
+     
+
+        StringBuilder s = new StringBuilder();
```

Comment:
have proper variable names. dont name it like i, s. Have names in such a way that seeing the variable name it should be clear